server/vespa/reindex.sh

```
@@ -0,0 +1,80 @@
+#!/bin/sh
+set -e
+
+
+# Get document type from user
+read -p "Enter document types to reindex (e.g., mail,file,event etc): " documentTypes
+
+if ! command -v jq &> /dev/null; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, &> is undefined.

(SC3020)